### ISSUE TITLE
Abnormal transferrin concentration

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6619]
**Preferred term label:** Abnormal transferrin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormality of iron homeostasis HP:0011031

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
These transferrin terms already exist, see https://hpo.jax.org/app/browse/term/HP:0032385
